src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -312,8 +319,11 @@ const Contact: React.FC<Props> = ({
                           <Typography variant="subtitle2">
                             {`Current: ${getLocalizedContactStatus(
                               t,
-                              status as StatusEnum,
-                            )} ${
+                              currentStatus,
+                            )}`}
+                          </Typography>
+                          <Typography variant="subtitle2">
+                            {`${
                               amount && amountCurrency
                                 ? currencyFormat(amount, amountCurrency, locale)
                                 : ''
```

Comment:
I think using a template string just adds noise. I'd write this as 

```
{amount && amountCurrency ? currencyFormat(amount, amountCurrency, locale) : null}
{getLocalizedPledgeFrequency(t, frequencyValue as PledgeFrequencyEnum)}
```

Reply:
I needed the template string to add the space between the pledgeAmount and the frequency. Any other thoughts on how I could do that?

Reply:
Gotcha. `{expression1}{' '}{expression2}` is how to do that since prettier will trim off the trailing space at the end of a line.